src/index.js

```diff
@@ -1,36 +1,46 @@
-import './styles.css';
+import './styles/style.css';
+import addTrash from './modules/function.js';
 
-let taskList = [];
+const input = document.getElementById('input');
 
-function taskCreator(str) {
-  const task = {
-    description: `${str}`,
-    completed: false,
-    index: taskList.length,
-  };
-  taskList = taskList.concat(task);
-}
+const target = document.getElementById('list');
 
-taskCreator('wash the dishes');
-taskCreator('complete the To Do List Project');
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
 
-localStorage.taskListData = JSON.stringify(taskList);
+function saveToLocalStorage(data) {
+  tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  tasks.push(data);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
 
-const taskShelf = document.querySelector('#to-do-list__shelf');
+input.addEventListener('keyup', (event) => {
+  if (event.key === 'Enter') {
+    const inputValue = input.value;
```

Comment:
- [ ] Good job implementing a function for editing task descriptions 👍 , but currently we can't edit two descriptions and save them at the same time in local storage:
   - let's add two tasks `js` and `html`
   
![Screenshot 2023-06-21 203740](https://github.com/Bleu-Yves-Sopoude/ToDoList/assets/57408419/2f9abe98-e2a1-4d35-a392-4b58538138ec)

   -  Then let edit them to `javaScript` and `HTML5`

![Screenshot 2023-06-21 204129](https://github.com/Bleu-Yves-Sopoude/ToDoList/assets/57408419/5b9ff753-c2c9-4822-a604-0112c9eb3188)

  - Then let's refresh the page to see if we are saving the changes in the local storage 

![Screenshot 2023-06-21 204417](https://github.com/Bleu-Yves-Sopoude/ToDoList/assets/57408419/ccd0fe3e-74ee-42a4-8e13-8f02f90e26f5)

As you can see, only the last task has been saved `HTML5` 😃. Kindly, fix it 👍

Reply:
Addressed